### PR TITLE
Added grants for clickhouse_info

### PR DESCRIPTION
**Pull request by aleksvagachev**
##### SUMMARY
Added collection of all grants for users and roles

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: plugins/modules/clickhouse_info.py [hunk 1]
```diff
@@ -532,6 +539,47 @@ def get_quotas(module, client):
...
+             "table, column, is_partial_revoke, grant_option FROM system.grants")
+
+    result = execute_query(module, client, query)
+
+    if result == PRIV_ERR_CODE:
+        return {PRIV_ERR_CODE: "Not enough privileges"}
+
+    grants_info = {'user_name': {},
+                   'role_name': {},
+                   }
```
Comment (Andersson007) [inline plugins/modules/clickhouse_info.py:557]:
```suggestion
    grants_info = {
        'user_name': {},
        'role_name': {},
    }
```
How about using `users` and `groups` instead ?
If I understand it correctly those dicts can contain multiple users and groups, right?
It'd look like:
```
grants_info: {
    users: {
        bob: { ... },
        alice: { ... },
    },
   roles: {
       accountant: { ... },
       sales: { ... },
   },
}
```

File: plugins/modules/clickhouse_info.py [hunk 2]
```diff
@@ -161,6 +161,13 @@
   type: dict
   sample: { "storage_policies": "..." }
   version_added: '0.4.0'
+grants:
+  description:
+    - The content of the system.grants table with user_name and role_name names as keys.
```
Comment (Andersson007) [inline plugins/modules/clickhouse_info.py:166]:
```suggestion
    - The content of the system.grants table for users and roles as keys.
```

**Review by Andersson007 — COMMENTED** [review]
@aleksvagachev thanks for the PR!

**Comment by aleksvagachev** [comment]
@Andersson007 Corrected the comments. Initially, I used the column name as the name of the keys.

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@aleksvagachev thanks for the contribution!

**Pull request merged**